### PR TITLE
[Guide] Creating a Multichain Connection with Web3Auth and React PnP no-modal SDK

### PR DESCRIPTION
**Pull request by rtomas**
- Add guide for multiple chain connection

https://web3auth-docs-git-add-guide-multiple-chains-web3auth.vercel.app/guides/pnp-no-modal-multichain

### CODE REVIEW

File: src/pages/guides/pnp-no-modal-multichain.mdx [hunk 1]
```diff
@@ -0,0 +1,265 @@
+---
+title: Creating a Multichain Connection with Web3Auth and React PnP no-modal SDK
+image: "guides/banners/multi.png"
+description: Learn how to connect to different blockchain using Web3Auth.
+type: guide
+tags: [multichain, polkadot, evm, cosmos, no-modal]
+date: Febreaury 9, 2024
+author: Web3Auth Team
+order: 1
```
Comment (shahbaz17) [inline src/pages/guides/pnp-no-modal-multichain.mdx:9]: Please change the order for this guide, not sure we want this as our first guide. Also, change the guide order based on the date, and have a featured boolean label to display some guides as required.

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

**Review by shahbaz17 — CHANGES_REQUESTED** [review]
Left some comments on the preview. Please check once @rtomas

**Comment by shahbaz17** [comment]
> * Add guide for multiple chain connection
> * First, we have to merge (because of dependencies) : [[Guide] Build a MPC wallet in iOS #702](https://github.com/Web3Auth/web3auth-docs/pull/702) and then update this PR
> 
> https://web3auth-docs-git-add-guide-multiple-chains-web3auth.vercel.app/guides/pnp-no-modal-multichain

Why this is a dependency @rtomas ?

**Comment by rtomas** [comment]
> > * Add guide for multiple chain connection
> > * First, we have to merge (because of dependencies) : [[Guide] Build a MPC wallet in iOS #702](https://github.com/Web3Auth/web3auth-docs/pull/702) and then update this PR
> > 
> > https://web3auth-docs-git-add-guide-multiple-chains-web3auth.vercel.app/guides/pnp-no-modal-multichain
> 
> Why this is a dependency @rtomas ?

There is no need to approve that dependency

**Review by shahbaz17 — APPROVED** [review]
lgtm

**Pull request merged**